projects/bignum-fuzzer/fuzzer_openssl_go_no_negative_num_len_1200_all_operations.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 5000
```

Comment:
Do you really need this?

Reply:
The default 4096 bytes will not suffice because the submitted fuzzer is configured to extract 4 bignums of 1200 bytes each. It can infer this from the files in the corpus, though. 1200 * 4 = 4800 and then 200 bytes max for operation encoding (add, sub, ...).

Reply:
My preference is that the corpus has some appropriately large inputs. And then inside the fuzz target itself you can have `if(size > 5000) return 0` and drop the options file.

Reply:
Sure.

Reply:
FYI, we don't use libFuzzer's default value of 4096. By default, we don't pass any value in `max_len`, so it is derived from the corpus (libFuzzer takes length of the longest input). And in 33% of runs we pass a random value in range of [1, 10000].